src/app/shared/modules/selectors/faction-selector/faction-selector-modal.component.ts

```diff
@@ -1,19 +1,21 @@
 import { ChangeDetectionStrategy, Component } from '@angular/core';
 import { BsModalRef } from 'ngx-bootstrap/modal';
 
-import { FactionSearchService } from '../../search/faction-search.service';
 import { Faction, FACTION_ID } from '@keira-types/faction.type';
+import { FactionSearchService } from '../../search/faction-search.service';
 import { SearchSelectorModalComponent } from '../base-selector/search-selector-modal.component';
 
 @Component({
   changeDetection: ChangeDetectionStrategy.OnPush,
   selector: 'keira-faction-selector-modal',
   templateUrl: './faction-selector-modal.component.html',
-  styleUrls: ['./faction-selector-modal.component.scss'],
 })
 export class FactionSelectorModalComponent extends SearchSelectorModalComponent<Faction> {
   /* istanbul ignore next */ // because of: https://github.com/gotwarlost/istanbul/issues/690
-  constructor(protected bsModalRef: BsModalRef, public searchService: FactionSearchService) {
+  constructor(
+    protected bsModalRef: BsModalRef,
+    public searchService: FactionSearchService,
+  ) {
```

Comment:
same: better to use `inject`

Reply:
I do not know if I can do it considering the mother-class and the use of super() inside the constructor